### DISCUSSION TITLE
Why does trace.ignore_path not work for some api?

### DISCUSSION THREAD

**Discussion by gaiwey** [apache/skywalking#9025] [Q&A] [answered]
skywalking version is 8.9.1
skywalking java agent version is 8.9.0

For example
the correct endpoint: http://192.168.1.12:8080/context-path/actuator/prometheus
the bad endpoint:       http://192.168.1.12:8080/actuator/prometheus

When accessing the error endpoint prompts 404 error.
I added java agent config/apm-trace-ignore-plugin.config: trace.ignore_path=/actuator/**
but trace can still track this request,  what is the reason?

![image](https://user-images.githubusercontent.com/21032881/167561373-daa4af54-70d9-4435-af50-3be16ce6cbdd.png)

**Comment by wu-sheng** [comment, marked as answer]
`GET:` is a part of endpoint name too.

**Reply by gaiwey** [reply]
Why does the name start with "GET:"？

**Reply by wu-sheng** [reply]
Because some plugins say so, I just reply based on your screenshot. This is not a strict rule from the SkyWalking perspective.

**Reply by myithxq** [reply]
Is there a solution? I'm seeking help.
